slik_wrangler_app/pages/operations.py

```diff
@@ -2,7 +2,9 @@
 import docs
 
 import streamlit as st
-from streamlit.scriptrunner import script_run_context
+#from streamlit.scriptrunner import script_run_context
+from streamlit.runtime.scriptrunner.script_run_context import get_script_run_ctx
```

Comment:
@PhagoroyeBabs where did you use the `get_script_run_ctx` ?

Reply:
@AdesholaAfolabi Sorry i dont know my changes is affecting the file directly. I thought i was working locally but my code editor is synchronizing with this here.

Reply:
Actually i figured that the script_run_context is the one blocking it to run on my pc as i have an upgraded version of the streamlit the one used in this code might not work for streamlit 0.12 and 0.14 of which i have but get_script_run_ctx worked though i still have another error to solve but that of the above have been trashed out.